test/JDBC/expected/BABEL-3347-vu-cleanup.out

```diff
@@ -0,0 +1,26 @@
+DROP VIEW IF EXISTS babel_3347_vu1
```

Comment:
Here, `vu` means `version_upgrade`. So the naming convention would be `babel_3347_vu_v1`.